modules/lib/lib-app/src/main/resources/lib/xp/examples/app/list.js

```diff
@@ -0,0 +1,42 @@
+var appLib = require('/lib/xp/app');
+var assert = require('/lib/xp/testing');
+
+/* global log*/
+
+// BEGIN
+// List virtual apps.
+var result = appLib.list();
+
+log.info('Listed apps: ' + result.map(app => app.key).join(', '));
```

Comment:
Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/enonic/xp/pullRequest?prid=9736743)
```suggestion
log.info('Listed apps: ' + result.map((app) => app.key).join(', '));
```